l10n/de/mobile/android/base/cliqz_strings.dtd

```diff
@@ -8,6 +8,9 @@
 <!-- Onboarding -->
 <!ENTITY firstrun_urlbar_subtext3 "&firstrun_urlbar_subtext;">
 
+<!-- Top Sites -->
+<!ENTITY activity_stream_empty_toppanel "Hier erscheinen deine meistbesuchten Seite">
```

Comment:
We do not add strings in the translations

Reply:
Removed this and fixed the conflict :+1: